src/styles/GoalPage.module.css

```diff
@@ -0,0 +1,53 @@
+.goalDescriptionContainer {
+  background-color: #ffffff;
```

Comment:
If you would, please standardize between one choice: "background" and "background -color" in this file and pull the colors in from Globals.css  for the colors on the format:
"background-color: var(--grey-100):"